templates/launch.html

```diff
@@ -160,21 +172,71 @@ <h3>Runner Data</h3>
-                <label for="eq_id">eq_id</label>
-                <input id="eq_id" name="eq_id" type="text" value="${eqIdValue}" class="qa-eq_id">
-            </div>
-            <div class="field-container">
-                <label for="form_type">form_type</label>
-                <input id="form_type" name="form_type" type="text" value="${formTypeValue}" class="qa-form_type">
-            </div>
-        `
+        document.querySelector('#survey_metadata_fields').style.display = "block";
+
+        if (launchPattern === "v1") {
+            document.querySelector('#survey_metadata_fields').innerHTML = `
+                <h3>${survey_type} Survey Metadata</h3>
+                <div class="field-container" class="visible-block">
+                    <label for="eq_id">eq_id</label>
+                    <input id="eq_id" name="eq_id" type="text" value="${eqIdValue}" class="qa-eq_id" >
+                </div>
+                <div class="field-container" class="visible-block">
+                    <label for="form_type">form_type</label>
+                    <input id="form_type" name="form_type" type="text" value="${formTypeValue}" class="qa-form_type">
+                </div>
+            `
+        } else {
+            document.querySelector('#survey_metadata_fields').innerHTML = `
+                <h3>${survey_type} Survey Metadata</h3>
+                <div class="field-container" class="visible-block">
+                    <label for="form_type">form_type</label>
+                    <input id="form_type" name="form_type" type="text" value="${formTypeValue}" class="qa-form_type">
+                </div>
+            `
+        }
+
+    }
+
+    function clearSchemaNameSelection() {
+        let schemaNameElement = document.querySelector("#schema_name")
+        schemaNameElement.selectedIndex = 0
+
+        let launchPattern = document.querySelector("#launch_pattern").value
+
+        if (launchPattern === "v1") {
+            document.querySelectorAll(".Social").forEach(function(element) {element.setAttribute("disabled", "disabled")})
+        } else {
+            document.querySelectorAll(".Social").forEach(function(element) {element.removeAttribute("disabled")})
+        }
+        
+        const schemaName = schemaNameElement.value
+
+        if (schemaName !== "Select Schema" && launchPattern === "v1") {
+            document.querySelector("#submit-btn-v1").style.display = 'inline-block';
+            document.querySelector("#submit-btn-v2").style.display = 'none';
+        }
+        else if (schemaName !== "Select Schema" && launchPattern === "v2") {
+            document.querySelector("#submit-btn-v2").style.display = 'inline-block';
+            document.querySelector("#submit-btn-v1").style.display = 'none';
+        }
+        else {
+            document.querySelector("#submit-btn-v2").style.display = 'none';
+            document.querySelector("#submit-btn-v1").style.display = 'none';
+        }
```

Comment:
To me something like this is more readable and tidier
```suggestion
        if (schemaName !== "Select Schema") {
            if (launchPattern === "v1") {
                document.querySelector("#submit-btn-v1").style.display = 'inline-block';
                document.querySelector("#submit-btn-v2").style.display = 'none';
            } else if (launchPattern === "v2") {
                document.querySelector("#submit-btn-v2").style.display = 'inline-block';
                document.querySelector("#submit-btn-v1").style.display = 'none';
            }
        } else {
            document.querySelectorAll("#submit-btn-v1", "#submit-btn-v2").style.display = 'none';
        }
```

Reply:
Updated this but had to use `querySelector` rather than `querySelectorAll` in the else